src/consensus/pool.rs

```diff
@@ -498,7 +498,10 @@ mod tests {
         assert!(!pool.is_notarized(0));
         for v in 0..11 {
             let vote = Vote::new_notar(0, Hash::default(), &sks[v as usize], v);
-            assert_eq!(pool.add_vote(vote).await, Ok(()));
+            match pool.add_vote(vote).await {
```

Comment:
There are no changes to production code in this PR.  So is it the case that we expect one of two different things to happen here (`Ok(())` or `Err(AddVoteError::SlotOutOfBounds)`).  But so far all the tests only trigger the `Ok(())`?